doctrine/doctrine-bundle/1.6/manifest.json

```diff
@@ -8,9 +8,9 @@
     },
     "env": {
         "#1": "Format described at https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url",
-        "#2": "For an SQLite database, use: \"sqlite:///%kernel.project_dir%/var/data.db\"",
+        "#2": "For an SQLite database, use: \"sqlite://%kernel.project_dir%/var/data.db\"",
```

Comment:
should be reverted, otherwise it doesn't work, I don't remember why though ;)

Reply:
reverted